Packs/Cyberint/Integrations/Cyberint/Cyberint.py

```diff
@@ -504,20 +540,53 @@ def fetch_incidents(client: Client, last_run: Dict[str, int],
                         attachments.append(tmp_attachment)
 
         alert["attachments"] = attachments
-        alert_csv_id = alert.get('alert_data', {}).get('csv', {}).get('id', '')
-        if alert_csv_id:
-            extracted_csv_data = extract_data_from_csv_stream(client, alert_id,  # type: ignore
-                                                              alert_csv_id)
-            alert['alert_data']['csv'] = extracted_csv_data
+        incident = {}
+
+        incident_csvs = dict_safe_get(alert, ['alert_data', 'csv', 'content']) or []
+
+        if not fetch_assign_multiple:
+            if incident_csvs:
+                index = 1
+                for incident_csv in incident_csvs:
+                    alert.update({'ref_id': f'{alert_id} ({index})'})
+                    alert['alert_data']['csv'].update({'content': incident_csv})

```

Comment:
Are we sure here that incident_csv is a record inside a single csv file and not a complete file?
In addition, it seems that the csv alert attribute is used as **file** and here it's a raw data. This is might be problematic since it breaks backward compatibility: what happens if another client of cyberint is fine with csv as a file and not raw data? his automations will fail!
Let's consider using another name instead of csv.

Reply:
We are sure that incident_csv is a record inside a single CSV file and not a complete file.
About the second part - I didn't get you.